src/molecules/Modal/__tests__/modal.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// import sinon from 'sinon';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import Modal from 'molecules/Modal';
+
+describe('<Modal />', () => {
+  // let defaultProps;
+  //
+  // beforeEach(() => {
+  //   defaultProps = {
+  //     // Add default props here
+  //   };
+  // });
```

Comment:
Also, you might consider adding a test to make sure that all of the props get correctly passed to the ReactModal